22:31:43 - INFO - eir.train_utils.utils - Global random seed set to 0.
Traceback (most recent call last):
  File "/home/panwei/he000176/.conda/envs/eir-gln/bin/eirtrain", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/panwei/he000176/.conda/envs/eir-gln/lib/python3.11/site-packages/eir/train.py", line 65, in main
    configs = get_configs()
              ^^^^^^^^^^^^^
  File "/home/panwei/he000176/.conda/envs/eir-gln/lib/python3.11/site-packages/eir/setup/config.py", line 114, in get_configs
    output_folder, log_level = get_output_folder_and_log_level_from_cl_args(
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/panwei/he000176/.conda/envs/eir-gln/lib/python3.11/site-packages/eir/setup/config.py", line 148, in get_output_folder_and_log_level_from_cl_args
    with open(config, "r") as f:
         ^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'configs/global_config.yaml'
